packages/react/src/elements/components/Flyout.js

```diff
@@ -4,24 +4,6 @@ import { Flyout as VanillaFlyout } from "hig-vanilla";
 import FlyoutAdapter from "../../adapters/FlyoutAdapter";
 
 class Flyout extends Component {
-  static propTypes = {
```

Comment:
Temporarily moving away from static definitions since they're not recognized by the plugin out of the box.